pipelines/controller/config/application.yaml

```diff
@@ -41,7 +41,11 @@ fhirdata:
   # "/<baseDirPath>/<prefix>" and can be absolute or relative. <baseDirPath>
   # is required for *nix, and must contain 1 or more directory names.
   # dwhRootPrefix: "gs://fhir-test-analytics/config/controller_DWH_ORIG"
-  dwhRootPrefix: "config/controller_DWH_ORIG"
+  #
+  # Note for developers: You can make a symlink of `[repo_root]/docker/dwh` here
+  # such that the Thrift Server of `compose-controller-spark-sql-single.yaml`
+  # config can easily be used in dev env. too.
+  dwhRootPrefix: "dwh/controller_DWH_DEV"
```

Comment:
Query : Just to get clarified, the symlink name will be `dwh`, should be created in this directory (`[repo_root]/pipeline/controller`) and points to `[repo_root]/docker/dwh`.